virttest/utils_libvirt/libvirt_nwfilter.py

```diff
@@ -28,3 +29,21 @@ def clean_up_nwfilter_binding(ignore_status=False):
     logging.info("existed nwfilter binding list is: %s", result)
     for binding_uuid in result:
         virsh.nwfilter_binding_delete(binding_uuid, ignore_status=ignore_status)
+
+
+def get_nwfilter_list():
+    """
+    Get nwfilter list in tuple(UUID, Name)
+
+    Usage:
+         # virsh nwfilter-list
+         UUID                                  Name
+         --------------------------------------------------
+         5c79e80b-cfb5-46d0-9490-77db3318a4b5  allow-arp
+         d8150f0b-2859-4049-a58d-963f33c59aa4  allow-dhcp
```

Comment:
This is the usage of virsh nwfilter-list, not the usage of this function. Would you please update with the actual usage, or remove this part?

Reply:
@chloerh 
Add more explanation for usage

Reply:
Please ignore my comment. Thanks.